### DISCUSSION TITLE
Please tell me your favorite music♪ あなたの好きな音楽を教えてください♪

### DISCUSSION THREAD

**Discussion by KokiOkai** [NAIST-SE/AttractingContributors#541] [General]
Please tell me your favorite music♪
あなたの好きな音楽を教えてください♪

My repository is here.
https://github.com/KokiOkai/AttractingContributors_Music

**Comment by ani-hovhannisyan** [comment]
I contributed #6 to your project, please can you do it back to mine?
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git
